src/test/resources/dependency-overwrite/validation/NullVersionTest.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: app
+version: 0.0.1
+
+dependencies:
+  - name: testDependency1
```

Comment:
S: One additional test case probably worth adding is the YAML  extended list format. (See [example 2.4](https://yaml.org/spec/1.2.2/#21-collections))
So something like:
```
dependencies:
  -
    name: testDependency1
    version: 0.0.1
    repository: https://fake.repo/
  -
    name: testDependency2
    version: 0.0.1
    repository: https://fake.repo/